desktop-widgets/importgps.cpp

```diff
@@ -0,0 +1,308 @@
+// SPDX-License-Identifier: GPL-2.0
+#include "ui_importgps.h"
+#include "desktop-widgets/importgps.h"
+#include "desktop-widgets/locationinformation.h"
+#include "core/dive.h"
+
+#include <time.h>
+#include <QFileDialog>
+#include <QProcess>
+#include <QDateTime>
+#include <QDate>
+#include <stdlib.h>
+#include <stdio.h>
```

Comment:
can we group the imports a little:

perhaps Q* imports at the bottom and C time/std* imports separated 

e.g.
```
#include "ui_importgps.h"
#include "desktop-widgets/importgps.h"
#include "desktop-widgets/locationinformation.h"
#include "core/dive.h"

#include <time.h>
#include <stdlib.h>
#include <stdio.h>

#include <QFileDialog>
#include <QProcess>
#include <QDateTime>
#include <QDate>
```

also aren't these imported from dive.h already?
```
#include <time.h>
#include <stdlib.h>
#include <stdio.h>
```

in general, if we can get away with not using C methods in this CPP file it would be beter.

Reply:
Done